test/regression/2015-08-05-symbolic-const-variable.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out -exit-on-error -exit-on-error-type=user %t.bc 2> %t.log
```

Comment:
> Your test is broken :) It will alway pass, because KLEE exits with the error that -exit-on-error-type=user is not a valid option.

Ops, what a typo! Well spotted, it took me 10 seconds to figure out the mistake. (Why isn't the type case insensitive?)

> Also, why do you use both -exit-on-error and -exit-on-error-type?

Well, how do you specify the error type? As pointed out by @delcypher, this test should not fail on every kind of error. Thinking about it more carefully, I would remove the `exit-on-error` and  check that an error is raised.

> And I would also add a valid klee_make_symbolic first.

Irrelevant in my opinion, since we already test the correct behavior of `klee_make_symbolic` in around 50 other tests.